tests/core/system-settings/task.yaml

```diff
@@ -22,5 +22,8 @@ execute: |
     timedatectl set-timezone "America/Lima"
     date +"%Z" | MATCH '\-05'
     timedatectl | MATCH "Time zone: America/Lima"
-    MATCH "America/Lima" < /etc/timezone
+    # /etc/timezone is missing on core24, skip check
```

Comment:
If /etc/timezone is redundant (looking at the symlink from /etc/localtime to /usr/share/zoneinfo), then perhaps the whole dance can be removed?

Reply:
Okay, let's drop the backup and the test for /etc/timezone and have fewer differences across core versions.

Reply:
Updated